app/controllers/patients_controller.rb

```diff
@@ -75,6 +65,9 @@ def patient_params
       :household_size_adults, :household_size_children, :insurance,
       :referred_by, :initial_call_date, :urgent_flag,
       :clinic_id,
+
+      :last_menstrual_period_days, :last_menstrual_period_weeks,
```

Comment:
Are the redundant with the same-name ones under `pregnancy` two lines down? Or should those be deleted? Or do I not understand what's happening here?